app/views/providers/check_provider_answers/index.html.erb

```diff
@@ -1,11 +1,11 @@
 <%= page_template(
-      page_title: t('.title'),
+      page_title: @read_only ? '' : t('.title'),
```

Comment:
Are you sure this page should have no title in read only? Should there be a read only version of the page title?

Reply:
Spoke to the designer and they said it would stay blank because they haven't come up with a title yet and they didn't want to block this ticket. There's also having different heading styles which will be designed later.